src/components/MaterialUI/Surfaces/ExpansionPanel.test.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { mount } from "enzyme";
+import ExpansionPanel from "./ExpansionPanel";
+import ExpansionPanelMui from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpansionPanelActions from "@material-ui/core/ExpansionPanelActions";
+import { ExpansionPanelProps, ExpansionPanelActionsProps } from "./expansionPanelProps";
+
+describe("Expansion Panel", () => {
+	it("Renders Expansion Panel", () => {
+		var header = <p>Header</p>;
+		var content = <p>Content</p>;
```

Comment:
Can we test multiple contents?  We will be using that for various components.

Reply:
What do you mean by multiple content? What wrapper makes is just puts passed header, content and actions in appropriate places. What we tested here is that he actually puts it in necessary places of MUI expansion panel. 

it will put components which are passed and the complexity of those components doesn't make any difference on how it will work.